### ISSUE TITLE
 tsup.egoist.sh resolves to malware

### ISSUE THREAD

**Issue by moltar** [egoist/tsup#1134]
I am not sure what the history is of the transition from `.sh` to `.dev`, but the old URLs that use `tsup.egoist.sh` redirect to a "badware" URLs.

![screenshot-20240601T112357-rWWVjyZY@2x](https://github.com/egoist/tsup/assets/491247/1090c101-667b-4d4b-a640-de5bde119c68)

**Comment by moltar** [comment]
There are still many places on GH that link to that URL: https://github.com/search?type=code&q=%22tsup.egoist.sh%22

**Comment by egoist** [comment]
yeah, i need to send pull requests to fix those links, do you know any bot that can do this? 🫠
